website/react-magma-docs/src/pages/api/dropdown.mdx

```diff
@@ -653,6 +653,11 @@ Dropdowns can contain elements other than `DropdownMenuItem` components. They ca
 While users may navigate between `DropdownMenuItem` components using up and down arrow keys,
 browser default keyboard behavior applies to other content.
 
+<Alert variant="info">
+  For displaying informational content or filters, consider using the{' '}
+  <Link to="/api/popover">Popover</Link> component instead of Dropdown.
+</Alert>
+
 ### Dropdown with Filter
```

Comment:
Lets get rid of the Dropdown with Filter + Dropdown with Informational Content examples, plus the added Popover ones. With this new alert, users should go over to the Popover page for those examples.